src/OpenTelemetry.Extensions.PersistentStorage.Api/OpenTelemetry.Extensions.PersistentStorage.Api.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <Description>OpenTelemetry Persistent Storage</Description>
+    <MinVerTagPrefix>Extensions.PersistentStorage.Api-</MinVerTagPrefix>
+  </PropertyGroup>
```

Comment:
Since it is a new project, might be nice to turn on:

```xml
    <Nullable>enable</Nullable>
    <EnableAnalysis>true</EnableAnalysis>
```

Also probably needs:

```xml
    <EnablePublicApi>true</EnablePublicApi>
```

Reply:
Running into some issue on this one. Will look in to this in separate PR.